snuba/cli/replacer.py

```diff
@@ -33,18 +33,21 @@
 @click.option('--log-level', default=settings.LOG_LEVEL, help='Logging level to use.')
 @click.option('--dogstatsd-host', default=settings.DOGSTATSD_HOST, help='Host to send DogStatsD metrics to.')
 @click.option('--dogstatsd-port', default=settings.DOGSTATSD_PORT, type=int, help='Port to send DogStatsD metrics to.')
-def replacer(replacements_topic, consumer_group, bootstrap_server, clickhouse_host, clickhouse_port, dataset,
+def replacer(replacements_topic, consumer_group, bootstrap_server, clickhouse_host, clickhouse_port, dataset_name: str,
```

Comment:
This changed from `dataset` to `dataset_name` to avoid making `dataset` a `Union[str, Dataset]`.

Reply:
Actually, I should probably split this out so it's consistent across all of these entrypoints (not that it's a big deal.)